metricbeat/module/postgresql/postgresql_test.go

```diff
@@ -38,55 +38,55 @@ func TestParseUrl(t *testing.T) {
 		{
 			Name:     "simple test",
 			URL:      "postgres://host1:5432",
-			Expected: "host=host1 port=5432",
+			Expected: "host='host1' port='5432'",
 		},
 		{
 			Name:     "no port",
 			URL:      "postgres://host1",
-			Expected: "host=host1",
+			Expected: "host='host1'",
 		},
 		{
 			Name:     "user/pass in URL",
 			URL:      "postgres://user:pass@host1:5432",
-			Expected: "host=host1 password=pass port=5432 user=user",
+			Expected: "host='host1' password='pass' port='5432' user='user'",
 		},
 		{
 			Name:     "user/pass in params",
 			URL:      "postgres://host1:5432",
 			Username: "user",
 			Password: "secret",
-			Expected: "host=host1 password=secret port=5432 user=user",
+			Expected: "host='host1' password='secret' port='5432' user='user'",
 		},
 		{
 			Name:     "user/pass in URL take precedence",
 			URL:      "postgres://user1:pass@host1:5432",
 			Username: "user",
 			Password: "secret",
-			Expected: "host=host1 password=pass port=5432 user=user1",
+			Expected: "host='host1' password='pass' port='5432' user='user1'",
 		},
 		{
 			Name:     "timeout no override",
 			URL:      "postgres://host1:5432?connect_timeout=2",
-			Expected: "connect_timeout=2 host=host1 port=5432",
+			Expected: "connect_timeout='2' host='host1' port='5432'",
 		},
 		{
 			Name:     "timeout from param",
 			URL:      "postgres://host1:5432",
 			Timeout:  3 * time.Second,
-			Expected: "connect_timeout=3 host=host1 port=5432",
+			Expected: "connect_timeout='3' host='host1' port='5432'",
 		},
 		{
 			Name:     "user/pass in URL take precedence, and timeout override",
 			URL:      "postgres://user1:pass@host1:5432?connect_timeout=2",
 			Username: "user",
 			Password: "secret",
 			Timeout:  3 * time.Second,
-			Expected: "connect_timeout=3 host=host1 password=pass port=5432 user=user1",
+			Expected: "connect_timeout='3' host='host1' password='pass' port='5432' user='user1'",
 		},
 		{
 			Name:     "unix socket",
 			URL:      "postgresql:///dbname?host=/var/lib/postgresql",
-			Expected: "dbname=dbname host=/var/lib/postgresql",
+			Expected: "dbname='dbname' host='/var/lib/postgresql'",
```

Comment:
Are these changes related/needed?

Reply:
Yep. Needed a newer version of the pg library with aix support, which in turn added single quotes for inline ssl support. See here: https://github.com/lib/pq/pull/818